lib/ProductOpener/Ingredients.pm

```diff
@@ -1398,6 +1398,23 @@ sub set_percent_max_values($$$) {
 			$changed++;
 		}
 
+		# For lists like  "Beans (52%), Tomatoes (33%), Water, Sugar, Cornflour, Salt, Spirit Vinegar"
+		# we can set a maximum on Sugar, Cornflour etc. that take into account that Water would be
+		# above that maximum
```

Comment:
not sure to understand. Above or below ?

Reply:
In this specific case, we want the percent max of Water to be set to 100 - 52 -33 = 15%, the max of sugar to be set to 15 / 2 = 7.5 %, the max of cornflour to be set to 15 / 3 etc. That's what this code does.